### PR TITLE
[TASK] Overhaul Backend UriBuilder page

### PR DESCRIPTION
**Pull request by linawolf**
In preparation for https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/940

The example from blog_example also need to be adjusted. Not part of this PR

Releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/Backend/UriBuilder.rst [hunk 1]
```diff
@@ -1,154 +1,128 @@
...
 :code:`\TYPO3\CMS\Backend\Routing\UriBuilder`.
 
-.. hint::
+..  hint::
 
    Make sure to use :code:`\TYPO3\CMS\Backend\Routing\UriBuilder` to create
    backend links and not :code:`\TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder`.
 
    The variable available as :code:`$this->uriBuilder` in a controller is the
    web routing UriBuilder and can only be used for frontend links.
```
Comment (froemken) [inline Documentation/ApiOverview/Backend/UriBuilder.rst:22]: The `hint` is indented with 4 spaces. Would be good to keep the following indented with 4 spaces, too.

```suggestion
    Make sure to use :code:`\TYPO3\CMS\Backend\Routing\UriBuilder` to create
    backend links and not :code:`\TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder`.

    The variable available as :code:`$this->uriBuilder` in a controller is the
    web routing UriBuilder and can only be used for frontend links.
```

**Review by froemken — CHANGES_REQUESTED** [review]

**Comment by froemken** [comment]
Hello @linawolf 

Quote: `The example from blog_example also need to be adjusted`

I don't get it as `columnsOnly` is currently not used in `blog_example`

Stefan

**Comment by linawolf** [comment]
> Quote: `The example from blog_example also need to be adjusted`
> 
> I don't get it as `columnsOnly` is currently not used in `blog_example`

No but the example is outdated in General (No DI, not using a valid view for the backend etc)

**Comment by linawolf** [comment]
See also https://github.com/TYPO3-Documentation/t3docs-examples/pull/276

**Review by froemken — APPROVED** [review]

**Pull request merged**